scripts/translation_pipeline.py

```
@@ -362,6 +373,48 @@ def get_jobs_for_document(self, document_id: str) -> List[TranslationJob]:
+    def add_quality_report(self, job_id: str, unit_id: str, quality_score: float,
+                     issues: List[str], suggestions: List[str], reviewer: str = "system") -> str:
+        """Add a quality report to a translation unit."""
+        report_id = str(uuid.uuid4())
+        
+        with self._lock:
+            job = self.translation_jobs.get(job_id)
+            if not job:
+                return ""
+            
+            # Use target_languages[0] instead of non-existent target_language
+            target_lang = job.target_languages[0] if job.target_languages else "unknown"
```

Flake8 (7.3.0)

[error] local variable 'target_lang' is assigned to but never used

(F841)